Makefile

```diff
@@ -84,7 +84,7 @@ start-e2e: # Ensure the PlacementDecision CRD is present for the ClusterDecision
 	kubectl apply -f https://raw.githubusercontent.com/open-cluster-management/api/a6845f2ebcb186ec26b832f60c988537a58f3859/cluster/v1alpha1/0000_04_clusters.open-cluster-management.io_placementdecisions.crd.yaml
 	NAMESPACE=argocd-e2e "dist/argocd-applicationset" --metrics-addr=:12345 --probe-addr=:12346 --argocd-repo-server=localhost:8081 --namespace=argocd-e2e
 
-# Begin the tests, targetting the standalone controller (started by make start-e2e) and the e2e argo-cd (started by make start-e2e)
+# Begin the tests, targeting the standalone controller (started by make start-e2e) and the e2e argo-cd (started by make start-e2e)
```

Comment:
Thanks, one of these days I will learn to correctly spell targeting, but not today :upside_down_face:.

Reply:
For me it's one of those words that I just spell twice and let spellcheck figure it out. 😆